docs/UserGuide.md

```diff
@@ -95,49 +63,36 @@ This command is used to add an employee to the address book. Let's examine how w
 
-* **What It Does:**
-  * Uses employee fields from the command to add an employee
-* **Format:**
-  * `add n/NAME p/PHONE_NUMBER e/EMAIL_ADDRESS a/HOME_ADDRESS [t/TAG]...`
-* **Examples:**
+### Adding a new employee
+When a new employee joins, you can use the `add` [command](#glossary) to add their information into HRMate.
 
-| S/N | Command information                                                                                                                                                                                                                                                                                                                                                                                                                            |
-|-----|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 1   | **Command:** `add n/John Doe p/98765432 e/johnd@example.com a/John Street, block 123 #01-01` <br><br> **Output:** Adds an employee with the following details into the address book: <ul><li>Name: John Doe</li> <li>Phone number: 98765432</li> <li>Email address: johnd@example.com</li> <li>Home address: John Street, block 123 #01-01</li></ul>                                                                                           |
-| 2   | **Command:** `add n/Betsy Crowe e/betsycrowe@example.com a/Newgate Prison p/91234567 t/criminal t/friend` <br><br> **Output:** Adds an employee with the following details into the address book: <ul><li>Name: Betsy Crowe</li> <li>Phone number: 91234567</li> <li>Email address: betsycrowe@example.com</li> <li>Home address: Newgate Prison</li> <li>Tags: criminal, friend (Note: we can add multiple tags to the same record)</li></ul> |
-* **Acceptable Values:**
-  * A non-empty input for name, phone number, email address, home address and each optional tag
-* **Expected Output on Success:**
-  * GUI Changes: the created user is shown in a dedicated interface section
-  * Message shown to user: "Successfully created employee"
-* **Expected Output on Failure:**
-  * `Missing argument error` -  if a compulsory argument is missing (compulsory arguments are `NAME`, `EMAIL`,
-    `PHONE_NUMBER` and `HOME_ADDRESS`)
-  * `FIELD must be of format...` - if at least one of the field inputs violates the field's format requirements. Follow
-    the displayed error message to fix the format error.
-* **For advanced users:**
-  * `add` takes in any number of tags, as an employee can have any number of tags
+#### Add an employee: `add`
```

Comment:
will it be better if the tenses of all titles are consistent? like "Adding an employee" instead of "Add an employee"

Reply:
changed